KsApi/lib/ApiDateProtocol.swift

```diff
@@ -0,0 +1,44 @@
+import Foundation
+
+public protocol ApiDateProtocol {
+  var date: Date { get }
+  func addingTimeInterval(_: TimeInterval) -> Self
+  init()
+  init(timeIntervalSince1970: TimeInterval)
+  init(timeIntervalSinceNow: TimeInterval)
+  var timeIntervalSince1970: TimeInterval { get }
+}
+
+extension Date: ApiDateProtocol {
+  public var date: Date {
+    return self
+  }
+}
+
+internal struct ApiMockDate: ApiDateProtocol {
+  private let time: TimeInterval
+
+  internal init() {
+    self.time = 1_475_361_315
```

Comment:
What does this number represent?

Reply:
Just a static timestamp so that the tests aren't affected by the device time.